geniza/corpus/admin.py

```diff
@@ -462,13 +463,48 @@ def get_urls(self):
     actions = (export_to_csv,)
 
 
+@admin.register(DocumentNeedsReview)
+class DocumentNeedsReviewAdmin(DocumentAdmin):
+    ordering = ("needs_review",)
+
+    list_display = (
+        "id",
+        "needs_review",
+        "shelfmark",
+        "description",
+        "doctype",
+        "all_languages",
+        "last_modified",
+        "is_public",
+    )
+
+    search_fields = (
+        "fragments__shelfmark",
+        "tags__name",
+        "description",
+        "notes",
+        "needs_review",
+        "id",
+        "old_pgpids",
+        "needs_review",
+    )
```

Comment:
search fields are ignored because we're using solr instead of django

Reply:
I removed these. Should we remove them from the other models as well?

Reply:
no, search fields needs to be defined for model admin to turn on search; I think there's a comment on document about this, and that's the only model that's affected